lambda/es-proxy-layer/lib/es_query.js

```diff
@@ -85,7 +91,7 @@ async function run_qid_query_es(params, qid) {
 
 function isQuestionAllStopwords(question) {
     let stopwords = "a,an,and,are,as,at,be,but,by,for,if,in,into,is,it,not,of,on,or,such,that,the,their,then,there,these,they,this,to,was,will,with".split(",");
-    let questionwords = question.toLowerCase().match(/\b(\w+)\b/g)
+    let questionwords = question.toLowerCase().split(" ")
```

Comment:
I think the proper way to do this is  question.toLowerCase().split(/\s+/) -- splitting on a space will result in multiple empty objects.  Not sure if that would be an issue, but matching on empty string could cause issues?